apps/src/sites/hourofcode.com/pages/public/index.js

```diff
@@ -123,8 +123,8 @@ $(document).ready(function () {
   });
 
   function checkShowCensusFollowUp() {
-    if ($("#twenty-hour-how-much").val() === "some" || $("#twenty-hour-how-much").val() === "all" || $("#ten-hour-how-much").val() === "some" ||
-    $("#ten-hour-how-much").val() === "all") {
+    if ($("#twenty-hour-how-much").val() === "SOME" || $("#twenty-hour-how-much").val() === "ALL" || $("#ten-hour-how-much").val() === "SOME" ||
```

Comment:
nit: break this long line up

Reply:
Agreed. From our [styleguide](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#all-languages):
> Stick to an 80 character line length when possible, except for long URLs in comments or long regular expressions. Occasionally wiggling between 80 and 120 characters is OK in cases where it greatly improves the readability over a multiple line split.